### PR TITLE
add 4.13 jobs for rosa-sts and remove 4.12 jobs

### PR DESCRIPTION
**Pull request by yasun1**

### CODE REVIEW

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-dev-preview-aws-rosa-sts-p2-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-dev-preview-aws-rosa-sts-p2-f7

**Comment by yasun1** [comment]
@yufchang @liangxia Please help review.

**Comment by yasun1** [comment]
@yufchang @liangxia Please help review.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -1139,19 +1139,6 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-workers-marketplace
-- as: aws-ocm-osd-ccs-p2-f7
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:1142]: Removal of OSD is expected, right ?

Comment (yasun1) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:1142]: And confirm that with @yufchang

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:1142]: Ack

**Pull request merged**